web/src/main/java/org/cbioportal/genome_nexus/web/IndexSearchController.java

```diff
@@ -0,0 +1,43 @@
+package org.cbioportal.genome_nexus.web;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+
+import org.cbioportal.genome_nexus.model.IndexSearch;
+import org.cbioportal.genome_nexus.service.IndexSearchService;
+import org.cbioportal.genome_nexus.web.config.InternalApi;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@InternalApi
+@RestController
+@CrossOrigin(origins="*")
+@RequestMapping(value= "/")
+@Api(tags = "Index-search-controller", description = "Index Search Controller")
+public class IndexSearchController {
+    private final IndexSearchService indexSearchService;
+
+    @Autowired
+    public IndexSearchController(IndexSearchService indexSearchService)
+    {
+        this.indexSearchService = indexSearchService;
+    }
+
+    @ApiOperation(value = "Performs index search.")
+    @RequestMapping(value = "/index/search",
```

Comment:
Not sure if we should inform programmers using the API that his searches the index. It seems very implementation specific. Maybe better to call it `/search` or if we group it with annotations: `/annotation/search`? What do you think?

Reply:
Change it to `/search`